sashimi-webapp/src/logic/documentPackager/index.js

```diff
@@ -1,31 +1,13 @@
-import markdownProcessor from './markdownProcessor';
-import xssFilter from './xssFilter';
+import work from 'webworkify-webpack';
 
-/**
-  * Convert markdown string into HTML string
-  * @param {string} markdown - can be mixed with html and css
-  * @return {Promise<string, error>} Promise - containing the HTML string
-  */
-const processMarkdown = (markdownString =>
-  new Promise((resolve, reject) => {
-    try {
-      resolve(markdownProcessor.process(markdownString));
-    } catch (error) {
-      reject(error);
-    }
-  })
-);
+const w = work(require.resolve('./worker.js'));
```

Comment:
Should the "require.resolve('./worker.js')" be changed to another import? E.g. "import worker from './worker.js'"???